packages/analytics/lib/index.d.ts

```diff
@@ -1675,6 +1705,42 @@ export namespace FirebaseAnalyticsTypes {
      */
     initiateOnDeviceConversionMeasurementWithEmailAddress(emailAddress: string): Promise<void>;
   }
+
+  /**
+   * Any custom event name string not in the standard list of recommended event names.
+   */
+  export declare type CustomEventName<T> = T extends EventNameString ? never : T;
+  /**
+   * Type for standard Google Analytics event names. logEvent also accepts any custom string and interprets it as a custom event name.
```

Comment:
Is there a URL we could place here to serve as a reference for the standard names (both for help maintaining in the future - these names change over time, even if slowly and we need to stay in sync) as well as for folks in the community (who may provide PRs in the future to update the event names, this has happened in the past)

Reply:
This type comes from [the docs](https://firebase.google.com/docs/reference/js/analytics.md#customeventname). The above doc comment is all that is mentioned in [the docs](https://firebase.google.com/docs/reference/js/analytics.md#customeventname)

Reply:
Hey @exaby73 - I wasn't clear, sorry. There are a huge number of strings here starting at line 1717, (example 'add_payment_info').

Where did they come from? I'd like a link to the list of things in EventNameString so that future maintainers will have a reference when trying to keep them in sync.

The question to ask your future self one year from now is: "Are these names still the valid names?", and the action to take is the action your future self wished your past self would take: that is, leave your future self an easy way to complete the inevitable task, while you are here working on it now...

Reply:
@exaby73 no response here. I need a response here - I'm expecting to see some link to some source of truth for this list so it may be compared now and in the future for correctness / freshness

Reply:
https://github.com/invertase/react-native-firebase/pull/7109/commits/6d0f422cd0f017ba82fd97a11702d949fcaf48f1

Reply:
Yes! Thank you